### PR TITLE
DevKit updates for master branch

### PR DESCRIPTION
**Pull request by ElectricMaxxx**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,20 +13,25 @@
     ],
     "require": {
         "php": "^5.6|^7.0",
```
Comment (dbu) [inline composer.json]: we stopped testing with 5.6 - this needs to be adjusted as well

Comment (ElectricMaxxx) [reply composer.json]: Oh yes. This are things i wanted to fix in this second round

Comment (ElectricMaxxx) [reply composer.json]: at least this should be compatible with travis checks at the end. I think about it ...

Comment (ElectricMaxxx) [reply composer.json]: https://github.com/symfony-cmf/block-bundle/pull/279 for now.

**Pull request merged**